setup.py

```diff
@@ -37,4 +37,8 @@ def write_version_py():
         ]
     },
     packages=find_packages(),
+    install_requires=[
+        "force_bdss >= 0.2.0",
+        "granta"
```

Comment:
Is it checking if the plugin is already installed here?

Reply:
no, this is their own library. We can't distribute that because it's closed source, so we just check for the dependency.